src/uiTest/scala/com/virtuslab/gitmachete/uitest/RunningIntelliJFixtureExtension.scala

```diff
@@ -84,26 +115,121 @@ trait RunningIntelliJFixtureExtension extends RobotPluginExtension { this: IdePr
 
       def checkoutBranch(branch: String): Unit = {
         runJs(s"project.checkoutBranch('$branch')")
-        intelliJ.probe.await()
+        probe.await()
       }
 
       def configure(): Unit = {
         runJs("project.configure()")
       }
 
+      def moveMouseToTheMiddleAndWait(secondsToWait: Int): Unit = {
+        runJs(s"project.moveMouseToTheMiddleAndWait('$secondsToWait')");
+      }
+
+      def switchRepo(idx: Int): Unit = {
+        runJs(s"project.findComboBoxAndSwitchRepo('$idx')")
+      }
+
+      object contextMenu {
+
+        def openContextMenu(branch: String): Unit = {
+          runJs(s"project.findCellAndRightClick('$branch')")
+          probe.await()
+        }
+
+        def checkout(): Unit = {
+          runJs(s"project.contextMenu.checkout()")
+          probe.await()
+        }
+
+        def checkoutAndSyncByRebase(): Unit = {
+          runJs(s"project.contextMenu.checkoutAndSyncByRebase()")
+          // Consciously not calling probe.await() after triggering rebase action. The method freezes UI
+          // and makes impossible to proceed with actions (e.g. findAndClickButton('Start Rebasing')).
+          // This comment applies here, below and to project level counterparts.
+        }
+
+        def syncByRebase(): Unit = {
+          runJs(s"project.contextMenu.syncByRebase()")
+        }
+
+        def checkoutAndSyncByMerge(): Unit = {
+          runJs(s"project.contextMenu.checkoutAndSyncByMerge()")
+          probe.await()
```

Comment:
Actually... this final `probe.await()` is so common that it's worth opening a good first issue for making it into `doAndAwait { ... }`